webClient/src/assets/i18n/log/messages_en.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Ignore the following for cluster support testing 

(This is testing for the logger messages substitution due to my unclean branch)